src/index.js

```diff
@@ -61,18 +62,18 @@ function clearCompleted() {
   renderList();
 }
 
-domObject.clearCompletedButton.addEventListener("click", () => {
+domObject.clearCompletedButton.addEventListener('click', () => {
   clearCompleted();
 });
 
-domObject.ul.addEventListener("click", (e) => {
+domObject.ul.addEventListener('click', (e) => {
   if (e.target) {
-    if (e.target.type === "checkbox") {
-      e.target.parentElement.classList.toggle("checked");
+    if (e.target.type === 'checkbox') {
+      e.target.parentElement.classList.toggle('checked');
       list[e.target.dataset.index].completed = e.target.checked;
       updateLocal();
     }
-    if (e.target.tagName === "I") {
+    if (e.target.tagName === 'I') {
       const numericIndex = Number(e.target.dataset.index);
       list = list.filter((item) => item.index !== numericIndex);
       index = list.length;
```

Comment:
- [ ] Kindly add a test for this statement in your group of tests. This `if` statement is responsible for deleting a single task from the list. Remember that one of the requirements as seen below is to mock HTML to test if the add or remove function works properly:

![image](https://user-images.githubusercontent.com/77106962/159122883-98a511ea-bf18-4884-bbc6-b98b3580c4a0.png)

Kindly implement this test.